code/src/App.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react'
+import React from "react";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+import { MovieList } from "pages/MovieList";
+import { MovieDetail } from "pages/MovieDetail";
+import "css/MovieList.css";
+import "css/MovieDetails.css";
+import "css/LinkButton.css";
+import "css/Navbar.css";
```

Comment:
Tip! Have a main css file where you include every part and then just include that one here. It will make it more clean